Source/bmalloc/libpas/src/libpas/pas_try_reallocate.h

```diff
@@ -364,6 +363,12 @@ pas_try_reallocate(void* old_ptr,
             allocate_callback, allocate_callback_arg);
         
         if (result.begin || free_mode == pas_reallocate_free_always) {
+            // Deallocate old PGM entry if its the one reallocated to other entry
+            bool pgm_return = pas_try_deallocate_pgm_large(old_ptr, config.config_ptr);
+            if (pgm_return) {
```

Comment:
Same change as previous comment (don't need `pgm_return` variable):

```suggestion
            if (pas_try_deallocate_pgm_large(old_ptr, config.config_ptr)) {
```

Reply:
Fixed